README.md

```diff
@@ -11,8 +11,15 @@
 
 ## Running
 
```

Comment:
Gradle should make it much easier to package/run this. Have you looked at packaging a `.gradlew` with your repo?